### PR TITLE
Allow to use write targets allocation via followers in proxying chunk service

### PR DESCRIPTION
**Pull request by gritukan**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

### CODE REVIEW

File: yt/yt/server/node/cluster_node/proxying_chunk_service.cpp [hunk 1]
```diff
@@ -270,20 +273,25 @@ class TProxyingChunkService
...
+        TAllocateWriteTargetsHandler(TProxyingChunkService* owner, bool useFollowers)
             : THandlerBase(owner)
+            , UseFollowers_(useFollowers)
         { }
 
     protected:
         TChunkServiceProxy::TReqAllocateWriteTargetsPtr CreateRequest() override
         {
-            return LeaderProxy_.AllocateWriteTargets();
+            auto proxy = UseFollowers_ ? LeaderProxy_ : FollowerProxy_;
```
Comment (ifsmirnov) [inline yt/yt/server/node/cluster_node/proxying_chunk_service.cpp:284]: Seems like it should be the other way.

File: yt/yt/server/node/cluster_node/proxying_chunk_service.h [hunk 2]
```diff
@@ -15,7 +15,8 @@ namespace NYT::NClusterNode {
 NRpc::IServicePtr CreateProxyingChunkService(
     NElection::TCellId cellId,
     TProxyingChunkServiceConfigPtr serviceConfig,
-    NApi::NNative::TMasterConnectionConfigPtr connectionConfig,
+    NApi::NNative::TMasterConnectionConfigPtr masterConnectionConfig,
+    NApi::NNative::TConnectionDynamicConfigPtr connectionConfig,
```
Comment (ifsmirnov) [inline yt/yt/server/node/cluster_node/proxying_chunk_service.h:19]: How about `nativeConnectionConfig` for better distinction?

**Review by ifsmirnov — CHANGES_REQUESTED** [review]

File: yt/yt/server/node/cluster_node/proxying_chunk_service.cpp (hunk 1 above)
Comment (gritukan) [reply yt/yt/server/node/cluster_node/proxying_chunk_service.cpp]: Oops, you are right!

File: yt/yt/server/node/cluster_node/proxying_chunk_service.h (hunk 2 above)
Comment (gritukan) [reply yt/yt/server/node/cluster_node/proxying_chunk_service.h]: Good idea!

**Comment by ifsmirnov** [comment]
Resolved: https://github.com/ytsaurus/ytsaurus/commit/c8141a87f252754efefe2cb6910d38a5177082ce

**Pull request closed without merging**